src/sentinel.c

```diff
@@ -552,10 +553,10 @@ void sentinelIsRunning(void) {
 
 /* Create a sentinelAddr object and return it on success.
  * On error NULL is returned and errno is set to:
- *  ENOENT: Can't resolve the hostname.
+ *  ENOENT: Can't resolve the hostname, unless accept_unresolved is non-zero.
  *  EINVAL: Invalid port number.
  */
-sentinelAddr *createSentinelAddr(char *hostname, int port) {
+sentinelAddr *createSentinelAddr(char *hostname, int port, int accept_unresolved) {
```

Comment:
I am not sure if  you could use a better variable name for the third argument: accept_unresolved?  I think it is very likely  represents a bool value, how about isXXX ir areXXX?

Reply:
thanks